Source/Core/DolphinWX/NetPlay/MD5Dialog.h

```diff
@@ -7,7 +7,6 @@
 #include <map>
 #include <string>
 #include <vector>
-#include <vector>
```

Comment:
Did clang-format remove that duplicate include, or did you do that by hand? If the former: cool!